screens/dashboard/index.js

```diff
@@ -1,13 +1,28 @@
 // @flow
 
-import React, {Component} from 'react';
-import {Alert, TouchableOpacity, StyleSheet, Text, TextInput, View} from 'react-native';
+import React, { Component } from 'react';
 import {bindActionCreators} from 'redux';
+import { 
+    StyleSheet, 
+    Text, 
+    View, 
+    TouchableHighlight, 
+    Animated, 
+    Alert, 
+    TouchableOpacity, 
+} from 'react-native';
 import {connect} from 'react-redux';
-import commonStyles from '../../styles/common';
+import { ScrollView } from 'react-native-gesture-handler';
+import { LinearGradient } from 'expo';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+// import global actions
 import * as actions from './actions';
 
-const styles = StyleSheet.create(commonStyles);
+// import global styles
+// @TODO: move the global styles from this screen into ../../styles/common 
```

Comment:
Note, we should clean up the styling so that the global styles are in `styles/common` and the local styles stay in this file.  But I'll create a new issue for cleaning that up.